cli/pcluster/models/hit/hit_cluster_model.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+from botocore.exceptions import ClientError
+
+from pcluster.cluster_model import ClusterModel
+from pcluster.config import mappings
+from pcluster.utils import get_instance_type
+
+
+class HITClusterModel(ClusterModel):
+    """HIT cluster model."""
+
+    def __init__(self):
+        super(HITClusterModel, self).__init__("HIT")
+
+    def get_cluster_section_definition(self):
+        """Get the cluster section definition used by the cluster model."""
+        return mappings.CLUSTER_HIT
+
+    def test_configuration(self, pcluster_config):
+        """Try to launch the requested instances (in dry-run mode) to verify configuration parameters."""
+        cluster_section = pcluster_config.get_section("cluster")
+        vpc_section = pcluster_config.get_section("vpc")
+
+        if not cluster_section or cluster_section.get_param_value("scheduler") == "awsbatch" or not vpc_section:
```

Comment:
Why there is no the condition `or cluster_section.get_param_value("cluster_type") == "spot"` we have in the SIT test?

Reply:
Because "cluster_type" is not present in HIT's cluster section.